client/src/components/Forms.js

```diff
@@ -38,8 +37,8 @@ export default Forms;
 
 const SignUpForm = (props) => {
   return ( 
-    <div>
-      <CardBody id="sign-up-body">
+    <div className="card-body" id="sign-up-body">
```

Comment:
card-body without a card surrounding it, it may be bringing in a lot of style stuff its not supposed to